### DISCUSSION TITLE
Problemas com o InteliCode no VsCode

### DISCUSSION THREAD

**Discussion by Guilherme-Sousa-Silva** [balta-io/forum#400] [Tira Dúvidas] [answered]
Em algumas ocasiões o IntelliCode não funciona no VsCode enquanto estou rodando códigos c#.
Estou usando as C# e C# Dev Kit

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Guilherme-Sousa-Silva, como vai?

Preciso de bem mais detalhes para poder te ajudar... tem algum print, algum erro?
